client/src/components/PinkHeader/PinkHeader.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import PinkHeader from './PinkHeader';
+
+storiesOf('PinkHeader', module)
+  .add("Search my Directory ", () => (
+    <PinkHeader 
+        placeholder="Search My Directory" 
+        type="search"/>
+  ))
+  .add("Book a Trip", () => (
+    <PinkHeader
+        bookTrip
+        placeholder="Search Trips"/>
+  ))
+  .add("Trip", () => (
+    <PinkHeader
+        Trip/>
+  ))
+  .add("Directory: My Friends", () => (
+    <PinkHeader
+        Friends
```

Comment:
Rather than thinking about this in terms of types of header (Trip Header, Friends Header, Groups Header, etc) think of this in terms of a modular header. We want all those combinations to work together